bin/runtime/Cargo.toml

```diff
@@ -157,14 +157,15 @@ runtime-benchmarks = [
 
 # Liminal-related features
 liminal = [
-    #"baby-liminal-extension",
```

Comment:
Ideally if we we'd check if those below two places
* https://github.com/Cardinal-Cryptography/aleph-node/blob/main/.github/workflows/_build-production-node-and-runtime.yml#L30
* https://github.com/Cardinal-Cryptography/aleph-node/blob/main/.github/workflows/_build-test-node-and-runtime.yml#L31

That both runtime and `aleph-node` binary does **not** have any liminal-related symbols

Reply:
checks added